tests/SpecTests/ClientSideEncryption/Prose22_RangeExplicitEncryptionTest.php

```diff
@@ -38,6 +38,8 @@ class Prose22_RangeExplicitEncryptionTest extends FunctionalTestCase
     private ?Client $encryptedClient = null;
     private $key1Id;
 
+    private static string $specDir = __DIR__ . '/../../specifications/source/client-side-encryption';
```

Comment:
Any reason not to use a `const` or `readonly` here? Doesn't make much difference -- just curious. The important thing is that it's concise to reference later.

Reply:
No real reason other than using `private static` in other places. Note that `readonly` wouldn't work, as `readonly` properties cannot have a default value but have to be initialised in the constructor.

I suppose we could change this property, as well as a number of other properties in UnifiedSpecTest private constants.